src/EPR.RegulatorService.Facade.API/Controllers/InviteRegulatorController.cs

```diff
@@ -170,7 +176,8 @@ public async Task<IActionResult> GetInvitedEnrollment([FromQuery] string email)
             }
             catch (Exception e)
             {
-                _logger.LogError($"Error when retriving the invite regulator token for user {User.UserId()}");
```

Comment:
The standard way to use Microsoft logger in other project is as follows
_logger.LogError($"Error when retrieving the invite regulator token for user {UserId}", User.UserId());